### PR TITLE
PEtab: Separate rdatas_to_simulation_df and rdatas_to_measurement_df for convenience

### LINKED ISSUE #972: bug in amici.petab_objective.get_edata_for_condition
```
~/PycharmProjects/amici-runs/venv/lib/python3.6/site-packages/amici/petab_objective.py in get_edata_for_condition(condition, problem_parameters, amici_model, petab_problem, observable_ids, parameter_mapping, parameter_scale_mapping, scaled_parameters)
    438                     "is not yet implemented.")
    439 
--> 440             species_idx = species_ids.find(species_id)
    441 
    442             if condition[PREEQUILIBRATION_CONDITION_ID]:

AttributeError: 'tuple' object has no attribute 'find'
```

### PR DESCRIPTION
**Pull request by MerktSimon**
Fixes #972 and #971

### CODE REVIEW

File: python/amici/petab_objective.py [hunk 1]
```diff
@@ -622,7 +623,8 @@ def _get_measurements_and_sigmas(
 def rdatas_to_measurement_df(
         rdatas: Sequence[amici.ReturnData],
         model: amici.Model,
-        measurement_df: pd.DataFrame) -> pd.DataFrame:
+        measurement_df: pd.DataFrame,
+        simulation: bool = False) -> pd.DataFrame:
```
Comment (dweindl) [inline python/amici/petab_objective.py]: I'd change that to string argument `measurement_column` with default value `petab.C.MEASUREMENT`.
Gives some more flexibility.

Comment (MerktSimon) [reply python/amici/petab_objective.py]: is there a way to set 2 "defaults" so users in interactive python can choose from a list, but still put in freehand?

Comment (dweindl) [reply python/amici/petab_objective.py]: not aware of it, but that doesn't mean much

Comment (FFroehlich) [reply python/amici/petab_objective.py]: you could create a second function `rdatas_to_simulation_df` that pipes all arguments to `rdatas_to_measurement_df` but has a different default for the column name

**Comment by FFroehlich** [comment]
Please add a basic testcase with/to one of the examples that petablints `rdatas_to_measurement_df`

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/petab_objective.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=5052164&bid=16633466)

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/petab_objective.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=5052164&bid=16633466)

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/petab_objective.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=5052164&bid=16633466)

**Comment by yannikschaelte** [comment]
what's the status on this? would be good to fix soon ...

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/petab_objective.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=5052164&bid=16633466)

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/petab_objective.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=5052164&bid=16633466)

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/petab_objective.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=5052164&bid=16633466)

**Comment by dweindl** [comment]
@MerktSimon : What is left to do here? Would be great to include that in a new release next week.

**Comment by dweindl** [comment]
> @MerktSimon : What is left to do here? Would be great to include that in a new release next week.

*ping*

**Pull request merged**